### PR TITLE
[DDO-1424] add sysbox installation chart

### PR DESCRIPTION
**Pull request by jack-r-warren**
https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-k8s.md#installation-of-sysbox but in a chart

### CODE REVIEW

File: charts/sysbox/templates/sysbox/sysbox-deploy-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+# CHANGES
```
Comment (choover-broad) [inline charts/sysbox/templates/sysbox/sysbox-deploy-rbac.yaml:2]: link to original manifests?

**Review by choover-broad — APPROVED** [review]
lgtm, thanks for this!

File: charts/sysbox/templates/sysbox/sysbox-deploy-rbac.yaml (hunk 1 above)
Comment (jack-r-warren) [reply charts/sysbox/templates/sysbox/sysbox-deploy-rbac.yaml:2]: good idea, linked in readme but files is a good idea

File: charts/sysbox/templates/sysbox/sysbox-deploy-k8s.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          sysbox-install: "yes"
+    spec:
+      serviceAccountName: sysbox-label-node
+      nodeSelector:
+        sysbox-install: "yes"
+      containers:
+      - name: sysbox-deploy-k8s
+        image: registry.nestybox.com/nestybox/sysbox-deploy-k8s
+        imagePullPolicy: Always
+        command: [ "bash", "-c", "/opt/sysbox/scripts/sysbox-deploy-k8s.sh install" ]
```
Comment (mflinn-broad) [inline charts/sysbox/templates/sysbox/sysbox-deploy-k8s.yaml:27]: `sysbox-deploy-k8s.sh` did you take a quick peek at what this script does if it's public, just to be safe?

**Review by mflinn-broad — APPROVED** [review]

File: charts/sysbox/templates/sysbox/sysbox-deploy-k8s.yaml (hunk 2 above)
Comment (jack-r-warren) [reply charts/sysbox/templates/sysbox/sysbox-deploy-k8s.yaml:27]: that's a good idea, I'll go find it

Comment (mflinn-broad) [reply charts/sysbox/templates/sysbox/sysbox-deploy-k8s.yaml:27]: can always docker exec into the image if it's not easily findable in the repos 😄

Comment (jack-r-warren) [reply charts/sysbox/templates/sysbox/sysbox-deploy-k8s.yaml:27]: Well, for a bash script, it is surprisingly well written, and it carves out special support for GKE oddities. It LGTM

**Pull request merged**